src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.RuntimeIdentifierInference.targets

```diff
@@ -9,7 +9,7 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
 Copyright (c) .NET Foundation. All rights reserved. 
 ***********************************************************************************************
 -->
-<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" InitialTargets="CheckRuntimeIdentifier">
```

Comment:
They used to be equivalent since the target we were running before was also Initial. But that was changed here: https://github.com/Microsoft/msbuild/pull/1280. To avoid the same class of issues, I'm making the call that validating only when the config validation is done is best. We were inconsistent between the runtime identifier check and the unknown target framework check so I standardized on the way that is consistent with the call made in  https://github.com/Microsoft/msbuild/pull/1280